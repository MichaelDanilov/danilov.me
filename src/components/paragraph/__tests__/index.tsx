import React from 'react';
import renderer from 'react-test-renderer';
import Paragraph from '..';

describe('Paragraph', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Paragraph
          paragraphs={[
            'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.',
            'Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.',
          ]}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
