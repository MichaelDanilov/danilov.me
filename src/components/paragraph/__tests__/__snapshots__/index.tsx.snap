// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph renders correctly 1`] = `
<div
  className="sc-bdfBwQ gSQAtd"
>
  <p
    className="sc-gsTCUz bGwQZY"
    dangerouslySetInnerHTML={
      Object {
        "__html": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
      }
    }
  />
  <p
    className="sc-gsTCUz bGwQZY"
    dangerouslySetInnerHTML={
      Object {
        "__html": "Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.",
      }
    }
  />
</div>
`;
